src/control/server/harness.go

```diff
@@ -244,17 +244,22 @@ func (h *IOServerHarness) Start(parent context.Context, membership *system.Membe
 		}
 	}
 
-	// now monitor them
-	select {
-	case <-parent.Done():
-		return parent.Err()
-	case err := <-errChan:
-		// If we receive an error from any instance, shut them all down.
-		// TODO: Restart failed instances rather than shutting everything
-		// down.
-		h.log.Errorf("instance error: %s", err)
-		if err != nil {
-			return errors.Wrap(err, "Instance error")
+	for {
+		select {
+		case <-parent.Done():
+			return parent.Err()
+		case err := <-errChan:
+			// If we receive an error from any instance, shut them all down.
+			// TODO: Restart failed instances rather than shutting everything
+			// down.
+			h.log.Errorf("instance exited: %v", err)
+			//			if err != nil {
+			//				return errors.Wrap(err, "Instance error")
+			//			}
```

Comment:
Should this be removed? Looks like instead of killing all instances (as noted above in the comment) now we're just logging some debug output.